tools/apigee-sackmesser/README.md

```diff
@@ -1,15 +1,18 @@
-# Apigee Portable Proxy Deployment Utility
+# Apigee Sackmesser (Swiss Army Knife)
 
-The Apigee portable deployment utility lets you deploy API proxies to Apigee
-Edge as well as hybrid/X without writing any additional configuration files. It
-can be used either as a commandline tool or a Docker container.
+The Apigee Sackmesser lets you deploy API proxies, shared flows and
```

Comment:
You could add a link from Sackmesser to a google image search or translate page if you want to!